"""fixed fk error in verses where it was bookId and not chapterId

Revision ID: 4c9b2703bdac
Revises: a9efb6712b2e
Create Date: 2021-11-03 09:56:18.771609

"""
from alembic import op
import sqlalchemy as sa
from sqlalchemy.dialects import mysql

# revision identifiers, used by Alembic.
revision = '4c9b2703bdac'
down_revision = 'a9efb6712b2e'
branch_labels = None
depends_on = None


def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.add_column('verse', sa.Column('chapterId', sa.Integer(), nullable=True))
    op.drop_constraint('verse_ibfk_1', 'verse', type_='foreignkey')
    op.create_foreign_key(None, 'verse', 'chapter', ['chapterId'], ['id'])
    op.drop_column('verse', 'bookId')
    # ### end Alembic commands ###


def downgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.add_column('verse', sa.Column('bookId', mysql.INTEGER(), autoincrement=False, nullable=True))
    op.drop_constraint(None, 'verse', type_='foreignkey')
    op.create_foreign_key('verse_ibfk_1', 'verse', 'chapter', ['bookId'], ['id'])
    op.drop_column('verse', 'chapterId')
    # ### end Alembic commands ###
